src/main/java/com/openlattice/chronicle/services/upload/AppDataUploadService.java

```diff
@@ -494,7 +505,12 @@ private void createEntitiesAndAssociations(
 
             String appPackageName, appName;
             appPackageName = appName = getFirstValueOrNull( appEntity, FULL_NAME_FQN );
-            String dateLogged = getMidnightDateTime( getFirstValueOrNull( appEntity, DATE_LOGGED_FQN ) );
+            String eventDate = getFirstValueOrNull( appEntity, DATE_LOGGED_FQN );
```

Comment:
Is this the same things Objects.firstNotNull or MoreObjects.firstNotNull?

Reply:
Nope. This gets returns the first value from a `SetMultimap<UUID, Object> entity` with key propertyTypeId(fqn)